src/haxe/net/impl/WebSocketJs.hx

```diff
@@ -31,6 +31,14 @@ class WebSocketJs extends WebSocket {
                 //haxe.io.Int8Array
                 //js.html.ArrayBuffer
                 trace('Unhandled websocket onmessage ' + m);
+            } else if (Std.is(m, js.html.Blob)) {

```

Comment:
You are right. I didn't notice the file. Sorry.
I will release 0.0.7 with your changes soon. Thanks!